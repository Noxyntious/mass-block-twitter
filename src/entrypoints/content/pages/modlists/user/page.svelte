<script lang="ts">
  import LayoutNav from '$lib/components/layout/LayoutNav.svelte'
  import { useRoute } from '$lib/components/logic/router'

  const route = useRoute()

  const userId = $derived(route.search?.get('userId'))

  onMount(() => {
    if (!userId) {
      return
    }
    $inspect(userId)
  })
</script>

<LayoutNav title="My Moderation Lists" />
<div>User {userId}</div>
