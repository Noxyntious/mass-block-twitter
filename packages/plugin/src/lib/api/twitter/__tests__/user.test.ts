import { describe, expect, it } from 'vitest'
import { parseBlockedUsers } from '../user'
import BlockedAccountsAll1 from './assets/BlockedAccountsAll1.json'
import BlockedAccountsAll2 from './assets/BlockedAccountsAll2.json'
import { omit } from 'es-toolkit'

describe('getBlockedUsers', () => {
  describe('parseBlockedUsers', () => {
    it('parseBlockedUsers page1', async () => {
      const users = parseBlockedUsers(BlockedAccountsAll1)
      expect(users.data).length(18)
      expect(users.cursor).not.undefined
      expect(users.cursor).toMatchSnapshot()
      expect(users.data.map((it) => omit(it, ['updated_at']))).toMatchSnapshot()
    })
    it('parseBlockedUsers page2', async () => {
      const users = parseBlockedUsers(BlockedAccountsAll2)
      expect(users.data).length(17)
      expect(users.cursor).not.undefined
      expect(users.cursor).toMatchSnapshot()
      expect(users.data.map((it) => omit(it, ['updated_at']))).toMatchSnapshot()
    })
  })
})
