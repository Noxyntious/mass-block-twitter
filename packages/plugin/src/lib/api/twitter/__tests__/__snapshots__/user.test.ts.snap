// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getBlockedUsers > parseBlockedUsers > parseBlockedUsers page1 1`] = `"6503585151335227393|1905470080662110188"`;

exports[`getBlockedUsers > parseBlockedUsers > parseBlockedUsers page1 2`] = `
[
  {
    "blocking": true,
    "created_at": "2025-03-14T18:30:39.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "libertis",
    "followers_count": 0,
    "following": false,
    "friends_count": 50,
    "id": "1900615488218202112",
    "is_blue_verified": false,
    "location": "",
    "name": "Townshend_arg",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900616452341657600/-IjTzFPI_normal.jpg",
    "screen_name": "Townshend_arg",
  },
  {
    "blocking": true,
    "created_at": "2025-03-14T11:26:02.000Z",
    "default_profile": true,
    "default_profile_image": true,
    "description": "",
    "followers_count": 1,
    "following": false,
    "friends_count": 65,
    "id": "1900508687950565376",
    "is_blue_verified": false,
    "location": "",
    "name": "NoHayK",
    "profile_image_url": "https://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png",
    "screen_name": "NoHayK220859",
  },
  {
    "blocking": true,
    "created_at": "2025-03-13T17:11:10.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Y sobre alas de gloria alza el pueblo
Trono digno a su gran majestad.",
    "followers_count": 21,
    "following": false,
    "friends_count": 220,
    "id": "1900233051478925312",
    "is_blue_verified": false,
    "location": "",
    "name": "The Real Argentinean Empire",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900239453416730624/kYspGOzG_normal.jpg",
    "screen_name": "TRAEmpire",
  },
  {
    "blocking": true,
    "created_at": "2025-03-11T19:49:08.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Licenciado en Finanzas
Intelectual de Miller
Ahorrista en pesos.
Argentina ser√° pr√≥spera
VLLC!!",
    "followers_count": 8,
    "following": false,
    "friends_count": 92,
    "id": "1899548109941673984",
    "is_blue_verified": false,
    "location": "Ciudad Aut√≥noma de Buenos Aire",
    "name": "Milton Gomez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1899548291789914112/XsUiw5As_normal.jpg",
    "screen_name": "MiltonGome56",
  },
  {
    "blocking": true,
    "created_at": "2025-03-11T01:39:30.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 6,
    "following": false,
    "friends_count": 126,
    "id": "1899273839252779008",
    "is_blue_verified": false,
    "location": "",
    "name": "Boris Umbert",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900052175532769280/hoBXoEuk_normal.jpg",
    "screen_name": "BorisUmber54052",
  },
  {
    "blocking": true,
    "created_at": "2025-03-07T12:28:51.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "gordo antikuka de C√≥rdoba",
    "followers_count": 3,
    "following": false,
    "friends_count": 27,
    "id": "1897987718870622208",
    "is_blue_verified": false,
    "location": "",
    "name": "gordo X antikuka",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1901027291502972928/izE1WLAu_normal.jpg",
    "screen_name": "gordomasterX",
  },
  {
    "blocking": true,
    "created_at": "2025-03-02T18:16:11.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 2,
    "following": false,
    "friends_count": 110,
    "id": "1896263242164441088",
    "is_blue_verified": false,
    "location": "Argentina",
    "name": "PardoVB",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1896611971648122880/A0Zxjc1t_normal.jpg",
    "screen_name": "PardoVB",
  },
  {
    "blocking": true,
    "created_at": "2025-02-26T18:48:32.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 4,
    "following": false,
    "friends_count": 56,
    "id": "1894821809478082564",
    "is_blue_verified": false,
    "location": "",
    "name": "Esteban Franchina",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1894821868755984384/o3TQM7Lc_normal.jpg",
    "screen_name": "efranchina",
  },
  {
    "blocking": true,
    "created_at": "2025-02-25T21:02:21.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 1,
    "following": false,
    "friends_count": 37,
    "id": "1894493073410752512",
    "is_blue_verified": false,
    "location": "",
    "name": "pedri gonzalez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1894493236489519104/tGRqOcEN_normal.jpg",
    "screen_name": "pedrigonza61876",
  },
  {
    "blocking": true,
    "created_at": "2025-02-25T03:56:03.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 14,
    "following": false,
    "friends_count": 9,
    "id": "1894234834685300736",
    "is_blue_verified": false,
    "location": "",
    "name": "Mauro Castagnaro",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1894234880462262273/TNSp47bW_normal.jpg",
    "screen_name": "CastagnaroMauro",
  },
  {
    "blocking": true,
    "created_at": "2025-02-23T13:52:35.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "El famoso hermano gemelo de tronco, su antitesis y su archienemigo",
    "followers_count": 211,
    "following": false,
    "friends_count": 235,
    "id": "1893660174570987520",
    "is_blue_verified": false,
    "location": "",
    "name": "Black Neura",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893663425743511552/_5MkihUH_normal.jpg",
    "screen_name": "0BlackNeura0",
  },
  {
    "blocking": true,
    "created_at": "2025-02-23T03:15:56.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Otro ciudadano cansado de tanto choreo. Banco la honestidad y la meritocracia siempre.
Anti-kuka desde siempre y para siempre.",
    "followers_count": 2,
    "following": false,
    "friends_count": 55,
    "id": "1893499855776923648",
    "is_blue_verified": false,
    "location": "Argentina",
    "name": "Los huevos al plato",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893501448471683072/Ox4xNdvg_normal.jpg",
    "screen_name": "lohuevosalplato",
  },
  {
    "blocking": true,
    "created_at": "2025-02-22T21:15:13.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "NO LA VES, MUGRE PERONCHA",
    "followers_count": 4,
    "following": false,
    "friends_count": 97,
    "id": "1893409102849523712",
    "is_blue_verified": false,
    "location": "Vaticano",
    "name": "MancuCiego",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893411052068495360/BLioEx68_normal.jpg",
    "screen_name": "MancuCiego",
  },
  {
    "blocking": true,
    "created_at": "2025-02-20T20:36:46.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 25,
    "following": false,
    "friends_count": 109,
    "id": "1892674699919753216",
    "is_blue_verified": false,
    "location": "",
    "name": "Fernando Gomez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892674782363340800/YfQBBdSG_normal.png",
    "screen_name": "gomez_nahu71389",
  },
  {
    "blocking": true,
    "created_at": "2025-02-20T14:05:31.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Comunicador. Hago videitos en internet.",
    "followers_count": 662,
    "following": false,
    "friends_count": 771,
    "id": "1892576259085123584",
    "is_blue_verified": true,
    "location": "Olivos",
    "name": "KENT",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892601355694391296/0HX9Qxe3_normal.jpg",
    "screen_name": "kentbrockmanxd",
  },
  {
    "blocking": true,
    "created_at": "2025-02-19T13:19:16.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "‚ò∫Ô∏èHola soy Joacoüá¶üá∑ ‚ù§Ô∏è, busco charlar y tener amistades OwO anime hentai y femboys claro q siii nwn   SIEMPRE BUENA ONDA PLSüò≥ü•∞   

 27yo NO MINORS pls",
    "followers_count": 34,
    "following": false,
    "friends_count": 133,
    "id": "1892202246554705920",
    "is_blue_verified": false,
    "location": "Yorha",
    "name": "üåü~joaquito Èóá„Å´Âí≤„ÅèÊúà~üåü",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900356300686368768/HQSQtWvs_normal.jpg",
    "screen_name": "TuAmigoJoaquito",
  },
  {
    "blocking": true,
    "created_at": "2025-02-18T23:52:36.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Argentinean Redneck  

- Dont Tread on Me -",
    "followers_count": 1,
    "following": false,
    "friends_count": 27,
    "id": "1891999074594779136",
    "is_blue_verified": false,
    "location": "",
    "name": "Mr. West",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892000197296758784/LxQTboYS_normal.jpg",
    "screen_name": "MisterWest1991",
  },
  {
    "blocking": true,
    "created_at": "2025-02-18T17:15:01.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Gordo reci√©n llegado a X para dar la batalla Cultural.",
    "followers_count": 64,
    "following": false,
    "friends_count": 201,
    "id": "1891899132907839489",
    "is_blue_verified": false,
    "location": "",
    "name": "Giga Termo",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892999953296826368/lpgBTU9__normal.jpg",
    "screen_name": "GigaTermoOrlado",
  },
]
`;

exports[`getBlockedUsers > parseBlockedUsers > parseBlockedUsers page2 1`] = `"6495636517327364096|1905470080662110169"`;

exports[`getBlockedUsers > parseBlockedUsers > parseBlockedUsers page2 2`] = `
[
  {
    "blocking": true,
    "created_at": "2025-02-18T03:22:32.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Mi opini√≥n es la que cuenta",
    "followers_count": 0,
    "following": false,
    "friends_count": 22,
    "id": "1891689672834793472",
    "is_blue_verified": false,
    "location": "",
    "name": "Tyler",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1891689796663197696/C8MFQx90_normal.jpg",
    "screen_name": "TylerAnCap",
  },
  {
    "blocking": true,
    "created_at": "2025-02-17T16:21:15.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Soldado del Javo. Macabeos 3:19",
    "followers_count": 6,
    "following": false,
    "friends_count": 34,
    "id": "1891523248694632457",
    "is_blue_verified": false,
    "location": "",
    "name": "Mileista de Milei",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1891528255636688896/PijNNjEF_normal.jpg",
    "screen_name": "FuerzaDelCieloX",
  },
  {
    "blocking": true,
    "created_at": "2025-02-16T21:18:50.000Z",
    "default_profile": true,
    "default_profile_image": true,
    "description": "",
    "followers_count": 4,
    "following": false,
    "friends_count": 245,
    "id": "1891235740430000128",
    "is_blue_verified": false,
    "location": "",
    "name": "Krinax",
    "profile_image_url": "https://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png",
    "screen_name": "Krinax77",
  },
  {
    "blocking": true,
    "created_at": "2025-02-16T18:31:30.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "The snake which cannot cast its skin has to die. As well the minds which are prevented from changing their opinions; they cease to be mind.¬†

üêçüêçüêç",
    "followers_count": 371,
    "following": false,
    "friends_count": 1959,
    "id": "1891193572399136768",
    "is_blue_verified": false,
    "location": "Argentina",
    "name": "Carrie Larrsen",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1903805306226040832/CqCAUNya_normal.jpg",
    "screen_name": "CarrieLarrsen09",
  },
  {
    "blocking": true,
    "created_at": "2025-02-16T11:18:11.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Vida, propiedad y libertad",
    "followers_count": 2,
    "following": false,
    "friends_count": 50,
    "id": "1891084596533751808",
    "is_blue_verified": false,
    "location": "",
    "name": "El Anarquista Paleolibertario",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893331876003102720/UZcuRC8V_normal.jpg",
    "screen_name": "paleoliberanarc",
  },
  {
    "blocking": true,
    "created_at": "2025-02-14T13:39:59.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Dios bendiga a la Republica Argentina. üá¶üá∑
üêç",
    "followers_count": 18,
    "following": false,
    "friends_count": 80,
    "id": "1890395529844084736",
    "is_blue_verified": false,
    "location": "",
    "name": "F.S.C‚öñÔ∏è",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1896750233532407808/JDkLBV-O_normal.jpg",
    "screen_name": "fsc0101",
  },
  {
    "blocking": true,
    "created_at": "2025-02-13T19:56:16.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "TENEMOS PRIORIDAD LOS DE LA DERECHA.",
    "followers_count": 14,
    "following": false,
    "friends_count": 67,
    "id": "1890127699597037568",
    "is_blue_verified": false,
    "location": "",
    "name": "Mileinardo üè¶",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1890131616770441216/xIi7_sJA_normal.jpg",
    "screen_name": "mileinardo",
  },
  {
    "blocking": true,
    "created_at": "2025-02-11T03:42:07.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 19,
    "following": false,
    "friends_count": 210,
    "id": "1889157898460332032",
    "is_blue_verified": false,
    "location": "Austria",
    "name": "Wensislao Quispe Mamani",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900302298389090304/qdHR-KV2_normal.jpg",
    "screen_name": "termotanquedsal",
  },
  {
    "blocking": true,
    "created_at": "2025-02-07T02:17:02.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Una ciudad sin villas miseria.",
    "followers_count": 10,
    "following": false,
    "friends_count": 113,
    "id": "1887686859595411456",
    "is_blue_verified": false,
    "location": "Buenos Aires",
    "name": "Porte√±os",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1887687793221971968/v9fVqotR_normal.jpg",
    "screen_name": "portenios36",
  },
  {
    "blocking": true,
    "created_at": "2025-02-06T20:51:55.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Me chamuy√© a Asado de Faso y la hermana// Ex Ej√©rcito, ex PFA. M√∫sico metalero, Contructor, T√©cnico PC e Intelectual de Miller //

AVE CHRISTUS REX - Lc 11:11",
    "followers_count": 70,
    "following": false,
    "friends_count": 259,
    "id": "1887604988060241923",
    "is_blue_verified": false,
    "location": "",
    "name": "XSICK",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1898506853115400193/RNA58Nkc_normal.jpg",
    "screen_name": "XSICK_777",
  },
  {
    "blocking": true,
    "created_at": "2025-02-04T22:38:20.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 47,
    "following": false,
    "friends_count": 147,
    "id": "1886907098950258688",
    "is_blue_verified": false,
    "location": "",
    "name": "Miller Intelectual ARG",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1902125379990499328/WDuAwK7S_normal.jpg",
    "screen_name": "Luciana1260052",
  },
  {
    "blocking": true,
    "created_at": "2025-02-04T04:46:01.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "SOY UN MASCAPITO",
    "followers_count": 1,
    "following": false,
    "friends_count": 28,
    "id": "1886637260507885568",
    "is_blue_verified": false,
    "location": "perukistan",
    "name": "Homero simpson",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1886637830035623936/SZBLs5EL_normal.jpg",
    "screen_name": "DelzarJuan66397",
  },
  {
    "blocking": true,
    "created_at": "2025-02-01T17:21:28.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 5,
    "following": false,
    "friends_count": 57,
    "id": "1885740144839524352",
    "is_blue_verified": false,
    "location": "",
    "name": "Nicol√°s Suarez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1886233978148958208/SEka133__normal.jpg",
    "screen_name": "Suarez89ok",
  },
  {
    "blocking": true,
    "created_at": "2025-02-01T12:01:21.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Me encargo de curar la ignorancia.

Si viniste a bloquarme, lo tuyo es terminal. Mis condolencias.",
    "followers_count": 45,
    "following": false,
    "friends_count": 117,
    "id": "1885659628924792832",
    "is_blue_verified": false,
    "location": "",
    "name": "Enojado An√≥nimo",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1885660103921008641/jnFiOGF1_normal.jpg",
    "screen_name": "EnojadoAnonimo",
  },
  {
    "blocking": true,
    "created_at": "2025-01-28T18:21:52.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "üá¶üá∑ Orden, progreso y libertad.",
    "followers_count": 99,
    "following": false,
    "friends_count": 210,
    "id": "1884305865815007234",
    "is_blue_verified": false,
    "location": "Argentina",
    "name": "Dark Voltaire",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1884792239995449344/2yrz8tUU_normal.jpg",
    "screen_name": "rar1021160",
  },
  {
    "blocking": true,
    "created_at": "2025-01-28T14:24:05.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "The Emperor Protects",
    "followers_count": 51,
    "following": false,
    "friends_count": 460,
    "id": "1884246012375552001",
    "is_blue_verified": false,
    "location": "",
    "name": "Primarch de Miller",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1884432290157109248/1kCbCngB_normal.jpg",
    "screen_name": "millerreborn",
  },
  {
    "blocking": true,
    "created_at": "2025-01-27T18:49:48.000Z",
    "default_profile": true,
    "default_profile_image": true,
    "description": "",
    "followers_count": 2,
    "following": false,
    "friends_count": 92,
    "id": "1883950498899976192",
    "is_blue_verified": false,
    "location": "",
    "name": "Lautaro",
    "profile_image_url": "https://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png",
    "screen_name": "lautarovollmer0",
  },
]
`;

exports[`getBlockedUsers > parseBlockedUsers page1 1`] = `"6503585151335227393|1905470080662110188"`;

exports[`getBlockedUsers > parseBlockedUsers page1 2`] = `
[
  {
    "blocking": true,
    "created_at": "2025-03-14T18:30:39.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "libertis",
    "followers_count": 0,
    "following": false,
    "friends_count": 50,
    "id": "1900615488218202112",
    "is_blue_verified": false,
    "location": "",
    "name": "Townshend_arg",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900616452341657600/-IjTzFPI_normal.jpg",
    "screen_name": "Townshend_arg",
  },
  {
    "blocking": true,
    "created_at": "2025-03-14T11:26:02.000Z",
    "default_profile": true,
    "default_profile_image": true,
    "description": "",
    "followers_count": 1,
    "following": false,
    "friends_count": 65,
    "id": "1900508687950565376",
    "is_blue_verified": false,
    "location": "",
    "name": "NoHayK",
    "profile_image_url": "https://abs.twimg.com/sticky/default_profile_images/default_profile_normal.png",
    "screen_name": "NoHayK220859",
  },
  {
    "blocking": true,
    "created_at": "2025-03-13T17:11:10.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Y sobre alas de gloria alza el pueblo
Trono digno a su gran majestad.",
    "followers_count": 21,
    "following": false,
    "friends_count": 220,
    "id": "1900233051478925312",
    "is_blue_verified": false,
    "location": "",
    "name": "The Real Argentinean Empire",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900239453416730624/kYspGOzG_normal.jpg",
    "screen_name": "TRAEmpire",
  },
  {
    "blocking": true,
    "created_at": "2025-03-11T19:49:08.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Licenciado en Finanzas
Intelectual de Miller
Ahorrista en pesos.
Argentina ser√° pr√≥spera
VLLC!!",
    "followers_count": 8,
    "following": false,
    "friends_count": 92,
    "id": "1899548109941673984",
    "is_blue_verified": false,
    "location": "Ciudad Aut√≥noma de Buenos Aire",
    "name": "Milton Gomez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1899548291789914112/XsUiw5As_normal.jpg",
    "screen_name": "MiltonGome56",
  },
  {
    "blocking": true,
    "created_at": "2025-03-11T01:39:30.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 6,
    "following": false,
    "friends_count": 126,
    "id": "1899273839252779008",
    "is_blue_verified": false,
    "location": "",
    "name": "Boris Umbert",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900052175532769280/hoBXoEuk_normal.jpg",
    "screen_name": "BorisUmber54052",
  },
  {
    "blocking": true,
    "created_at": "2025-03-07T12:28:51.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "gordo antikuka de C√≥rdoba",
    "followers_count": 3,
    "following": false,
    "friends_count": 27,
    "id": "1897987718870622208",
    "is_blue_verified": false,
    "location": "",
    "name": "gordo X antikuka",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1901027291502972928/izE1WLAu_normal.jpg",
    "screen_name": "gordomasterX",
  },
  {
    "blocking": true,
    "created_at": "2025-03-02T18:16:11.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 2,
    "following": false,
    "friends_count": 110,
    "id": "1896263242164441088",
    "is_blue_verified": false,
    "location": "Argentina",
    "name": "PardoVB",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1896611971648122880/A0Zxjc1t_normal.jpg",
    "screen_name": "PardoVB",
  },
  {
    "blocking": true,
    "created_at": "2025-02-26T18:48:32.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 4,
    "following": false,
    "friends_count": 56,
    "id": "1894821809478082564",
    "is_blue_verified": false,
    "location": "",
    "name": "Esteban Franchina",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1894821868755984384/o3TQM7Lc_normal.jpg",
    "screen_name": "efranchina",
  },
  {
    "blocking": true,
    "created_at": "2025-02-25T21:02:21.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 1,
    "following": false,
    "friends_count": 37,
    "id": "1894493073410752512",
    "is_blue_verified": false,
    "location": "",
    "name": "pedri gonzalez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1894493236489519104/tGRqOcEN_normal.jpg",
    "screen_name": "pedrigonza61876",
  },
  {
    "blocking": true,
    "created_at": "2025-02-25T03:56:03.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 14,
    "following": false,
    "friends_count": 9,
    "id": "1894234834685300736",
    "is_blue_verified": false,
    "location": "",
    "name": "Mauro Castagnaro",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1894234880462262273/TNSp47bW_normal.jpg",
    "screen_name": "CastagnaroMauro",
  },
  {
    "blocking": true,
    "created_at": "2025-02-23T13:52:35.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "El famoso hermano gemelo de tronco, su antitesis y su archienemigo",
    "followers_count": 211,
    "following": false,
    "friends_count": 235,
    "id": "1893660174570987520",
    "is_blue_verified": false,
    "location": "",
    "name": "Black Neura",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893663425743511552/_5MkihUH_normal.jpg",
    "screen_name": "0BlackNeura0",
  },
  {
    "blocking": true,
    "created_at": "2025-02-23T03:15:56.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Otro ciudadano cansado de tanto choreo. Banco la honestidad y la meritocracia siempre.
Anti-kuka desde siempre y para siempre.",
    "followers_count": 2,
    "following": false,
    "friends_count": 55,
    "id": "1893499855776923648",
    "is_blue_verified": false,
    "location": "Argentina",
    "name": "Los huevos al plato",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893501448471683072/Ox4xNdvg_normal.jpg",
    "screen_name": "lohuevosalplato",
  },
  {
    "blocking": true,
    "created_at": "2025-02-22T21:15:13.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "NO LA VES, MUGRE PERONCHA",
    "followers_count": 4,
    "following": false,
    "friends_count": 97,
    "id": "1893409102849523712",
    "is_blue_verified": false,
    "location": "Vaticano",
    "name": "MancuCiego",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1893411052068495360/BLioEx68_normal.jpg",
    "screen_name": "MancuCiego",
  },
  {
    "blocking": true,
    "created_at": "2025-02-20T20:36:46.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "",
    "followers_count": 25,
    "following": false,
    "friends_count": 109,
    "id": "1892674699919753216",
    "is_blue_verified": false,
    "location": "",
    "name": "Fernando Gomez",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892674782363340800/YfQBBdSG_normal.png",
    "screen_name": "gomez_nahu71389",
  },
  {
    "blocking": true,
    "created_at": "2025-02-20T14:05:31.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Comunicador. Hago videitos en internet.",
    "followers_count": 662,
    "following": false,
    "friends_count": 771,
    "id": "1892576259085123584",
    "is_blue_verified": true,
    "location": "Olivos",
    "name": "KENT",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892601355694391296/0HX9Qxe3_normal.jpg",
    "screen_name": "kentbrockmanxd",
  },
  {
    "blocking": true,
    "created_at": "2025-02-19T13:19:16.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "‚ò∫Ô∏èHola soy Joacoüá¶üá∑ ‚ù§Ô∏è, busco charlar y tener amistades OwO anime hentai y femboys claro q siii nwn   SIEMPRE BUENA ONDA PLSüò≥ü•∞   

 27yo NO MINORS pls",
    "followers_count": 34,
    "following": false,
    "friends_count": 133,
    "id": "1892202246554705920",
    "is_blue_verified": false,
    "location": "Yorha",
    "name": "üåü~joaquito Èóá„Å´Âí≤„ÅèÊúà~üåü",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1900356300686368768/HQSQtWvs_normal.jpg",
    "screen_name": "TuAmigoJoaquito",
  },
  {
    "blocking": true,
    "created_at": "2025-02-18T23:52:36.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Argentinean Redneck  

- Dont Tread on Me -",
    "followers_count": 1,
    "following": false,
    "friends_count": 27,
    "id": "1891999074594779136",
    "is_blue_verified": false,
    "location": "",
    "name": "Mr. West",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892000197296758784/LxQTboYS_normal.jpg",
    "screen_name": "MisterWest1991",
  },
  {
    "blocking": true,
    "created_at": "2025-02-18T17:15:01.000Z",
    "default_profile": true,
    "default_profile_image": false,
    "description": "Gordo reci√©n llegado a X para dar la batalla Cultural.",
    "followers_count": 64,
    "following": false,
    "friends_count": 201,
    "id": "1891899132907839489",
    "is_blue_verified": false,
    "location": "",
    "name": "Giga Termo",
    "profile_image_url": "https://pbs.twimg.com/profile_images/1892999953296826368/lpgBTU9__normal.jpg",
    "screen_name": "GigaTermoOrlado",
  },
]
`;
